src/components/LetterProofing.js

```diff
@@ -7,39 +7,61 @@ import { shortCodePattern } from "../login/app_utils/validation/regexPatterns";
 class LetterProofingButton extends Component {
   state = {
     letter_expired: false,
-    letter_expires: false,
+    letter_expires: "",
     verifyingLetter: false,
     confirmingLetter: false,
-    letter_sent: "",
```

Comment:
In the sprit of @fredrikt comments about naming, I wonder if these props could be a little more descriptive of what they pass on (i.e. that two are dates). I have also always wondered what the difference between the "confirming" prop and the "verifying" prop is here, maybe renaming could shed some light on it?